tests/unit/TableTest.php

```diff
@@ -274,27 +274,18 @@ public function testDeleteColumn()
         $this->assertNull($r);
     }
 
-    protected function createTestTable(string $tableName,bool $hasAge = true):Morebec\YDB\Table
+    protected function createTestTable(string $tableName,array $additionalColumns = []):Morebec\YDB\Table
     {
-        $schema = $this->createTestTableSchema($hasAge);
-
-        return $this->database->createTable(
-            new TableSchema($tableName,$schema)
-        );
-    }
-
-    protected function createTestTableSchema(bool $hasAge)
-    {
-        $schema = [
+        $baseSchema = [
             new Column('id', ColumnType::STRING(), true),
             new Column('first_name', ColumnType::STRING()),
             new Column('last_name', ColumnType::STRING()),
         ];
 
-        if ($hasAge) {
-            $schema[] = new Column('age', ColumnType::INTEGER());
-        }
+        $schema = array_merge($baseSchema,$additionalColumns);
```

Comment:
Same, here could you rename to $columns and $baseColumns

Reply:
Updates done